pkg/ibmvpc/examples/out/explain_out/NACLQueryConnection1_all_vpcs_explain_debug.txt

```diff
@@ -1,7 +1,8 @@
-Connectivity explanation for protocol: UDP between vsi1-ky and 161.26.0.0/16 within test-vpc1-ky
-================================================================================================
+Explaining connectivity from vsi1-ky to 161.26.0.0/16 within test-vpc1-ky using "protocol: UDP"
+===============================================================================================
+
+Connection from vsi1-ky[10.240.10.4] to Public Internet 161.26.0.0/16 using "protocol: UDP"
```

Comment:
```suggestion
Connections are allowed from vsi1-ky[10.240.10.4] to Public Internet 161.26.0.0/16 using "protocol: UDP"
```

Reply:
same answer as above

Reply:
my 2c: it means the same thing in both headers ("we are talking about a connection that has these properties") so I'd say it is preferable to use the same phrasing